config/code_schools.yml

```diff
@@ -11,6 +11,7 @@
 #  full_time:  Is your school full time? [true|false]
 #  hardware_included: Does tuition include hardware? [true|false]
 #  notes: <Addtional Comments> [optional]
+#  img: Image file name of logo
```

Comment:
The CSS automatically resizes it to fit and preserves aspect ratio, so I'm not sure if there's a size that would break it.. I updated to include location and format though.